mgmt/rest/client/task.go

```diff
@@ -93,6 +93,9 @@ func (c *Client) CreateTask(s *Schedule, wf *wmap.WorkflowMap, name string, dead
 // interactive with Event and Done channels. An HTTP GET request retrieves tasks.
 // StreamedTaskEvent returns if it succeeds. Otherwise, an error is returned.
 func (c *Client) WatchTask(id string) *WatchTasksResult {
+	// during watch we don't want to have a timeout
+	c.http.Timeout = time.Duration(0)
+
```

Comment:
Calling task watcher causes overwrite client timeout to 0 for ever. Maybe the timeout should be set to its real value (set by user) after finish task watch - what do you think about that?